generate_api_docs.py

```diff
@@ -38,8 +38,10 @@ def callable_to_source_link(obj):
     source = inspect.getsourcelines(obj)
     line = source[-1] + 1 if source[0][0].startswith("@") else source[-1]
     version = get_version(package_name)
+    version = version.replace("b", ".b")  # correct github links to beta releases
```

Comment:
This is not very pretty, would be glad to replace it with a less hacky solution

Reply:
One way to make this more robust would be to check if the link is valid via `urllib` as we run this script. @lgeiger how do you feel about this?

Reply:
That's fine, looks like PyPI names the release [1.0b2](https://pypi.org/project/larq-zoo/1.0b2/) which is different to the git tag